ro_help/hub/templates/ngo/list.html

```diff
@@ -142,68 +140,71 @@ <h2 class="ngo-list-title">
-                {% if need.resource_tags.all  %}
-                    <div class="tag-bar">
-                        {% for tag in need.resource_tags.all %}
-                            <span class="tag">{{ tag }}</span>
-                        {% endfor%}
+
+                <div class="card-info">
+                    <div class="flex-align-center">
+                        <div class="card-image need-logo is-hidden-tablet">
+                            <img src="{{ need.ngo.get_avatar }}" alt="{{ need.ngo.name }}">
+                        </div>
+                        <div class="need-title is-hidden-tablet">{{need.ngo.name}}</div>
                     </div>
-                {% endif %}
+                    <div class="need-title is-hidden-mobile">{{need.ngo.name}}</div>
+                    <div class="need-subtitle">{{ need.title|truncatechars:50 }}</div>
+                    {% if need.resource_tags.all  %}
+                        <div class="tag-bar">
+                            {% for tag in need.resource_tags.all %}
+                                <span class="tag">{{ tag }}</span>
+                            {% endfor%}
+                        </div>
+                    {% endif %}
 
-            </div>
-            <div class="vr"></div>
-            <hr class="is-hidden-tablet" style="margin: 0 10%;"/>
-            {% url 'ngo-helper-create' need.ngo.pk need.pk as base %}
+                </div>
+                <div class="vr"></div>
+                <hr class="is-hidden-tablet" style="margin: 0 10%;"/>
+                {% url 'ngo-helper-create' need.ngo.pk need.pk as base %}
+
+                <div class="need-call2action">
+                    <a class="button is-flex has-background-success has-text-weight-bold has-text-white"
+                       href="{% spurl base='{{ base }}' set_query='kind={{ need.kind }}' %}">
+                        {% if need.kind == "money" %}
+                            {% trans "Donate" %}
+                        {% else %}
+                            {% trans "I can help" %}
+                        {% endif %}
+                    </a>
+                </div>
 
-            <div class="need-call2action">
-                <a class="button is-flex has-background-success has-text-weight-bold has-text-white"
-                   href="{% spurl base='{{ base }}' set_query='kind={{ need.kind }}' %}">
-                    {% if need.kind == "money" %}
-                        {% trans "Donate" %}
-                    {% else %}
-                        {% trans "I can help" %}
-                    {% endif %}
-                </a>
             </div>
-
+            {% endfor %}
         </div>
-        {% endfor %}
-    </div>
-    {% include "shared/pagination.html" with page_obj=page_obj %}
+        {% include "shared/pagination.html" with page_obj=page_obj %}
+    {% else %}
+        <p style="text-align: center;">{% trans "No results matching query" %}</p>
```

Comment:
It look a little to crowded.
 
![Screenshot 2020-03-30 at 12 03 00](https://user-images.githubusercontent.com/639771/77894756-68ec8c00-727e-11ea-9b27-9d89ccec3ae4.png)

Maybe we can use something like bulma's container component https://bulma.io/documentation/layout/container/ or content https://bulma.io/documentation/elements/content/ to add some padding and maybe make the text a little bigger.